### DISCUSSION TITLE
Erro ao adicionar o método AddNotifications

### DISCUSSION THREAD

**Discussion by marcionevesdev** [balta-io/7196#25] [Perguntas e Respostas] [answered]
Bom dia galera. 
  O meu projeto não esta aceitando o método AddNotifications qual a versao do flunt que ele funciona ?
Desde já agradeço

**Comment by marcionevesdev** [comment, marked as answer]
Na aula nao é especificado a versão do flunt na instalação.  
Agora estou com esse problema e não tenho nenhum suporte.

**Reply by andrebaltieri** [reply]
Boa tarde @marcioLegiaodeAco como vai?

Sobre a versão do Flunt, ela pode ser vista no código fonte do projeto que está aberto aqui no GitHub:
https://github.com/balta-io/7196/blob/master/Todo.Domain/Todo.Domain.csproj

`<PackageReference Include="flunt" Version="1.0.4" />`

**Reply by marcionevesdev** [reply]
Obrigado , Funcionou !!
